packages/building-blocks/src/index.ts

```diff
@@ -1,5 +1,7 @@
 import blue from "./colors/blue";
 import red from "./colors/red";
 import grey from "./colors/grey";
+import { Dropdown } from "./dropdown";
 
+export { Dropdown };
 export default { blue, red, grey };
```

Comment:
hm, trying to visualize how this export interacts with the default export below it. Should it just be added to that?

Reply:
I feel like we should keep it as is since it is an individual component. Theres no need to package everything up as the default, especially as the building blocks package grows. Not sure what else is planned for building blocks, but it seems like it is going to keep growing.

Reply:
Yeah lets get rid of the default export. I can create a new otp-rr PR that is blocked by this PR or just fix the breaking changes in the otp-rr PR that goes with this PR.